Tests/MongoSwiftTests/ReadPreferenceTests.swift

```diff
@@ -3,6 +3,27 @@ import Nimble
 import TestsCommon
 import XCTest
 
+/// Indicates that a type has a read preference property, as well as a way to get a read preference from an instance
+/// of the corresponding mongoc type.
+private protocol ReadPreferenceable {
```

Comment:
somehow this sounds a lot worse than `ReadConcernable` but it's just for tests so I didn't spend too much time thinking about it lol